File: Python/Maria/Connection.py
import mariadb


def conectar():
    conn = mariadb.connect(
        host='SEU HOST',
        port=3306,
        user='SEU USER',
        password='SEU PASSWORD',
        database='SEU DATABASE',
    )

    return conn


def desconectar(conn):
    if conn:
        conn.close()

File: Julia/Big Query/Connection.jl
using BigQuery


function conectar()
    conn = BigQuery.connect("SEU PROJECT ID", "SEU DATASET ID", "SEU TABLE ID")

    return conn
end


function get_table(conn)
    table = conn |> BigQuery.get_table("SEU PROJECT ID", "SEU DATASET ID", "SEU TABLE ID")

    return table
end


function execute_query(conn, query)
    result = conn |> BigQuery.query(query)

    return result
end


function desconectar(conn)
    conn |> BigQuery.close()
end